requirements.txt

```diff
@@ -76,6 +76,7 @@ python_magic==0.4.25
 decorator==5.1.1
 blurhash==1.1.4
 python_dateutil==2.8.2
+types-python-dateutil==2.8.17
```

Comment:
`Docker/nab/requirements.txt` will need to be kept in sync.

Reply:
See #398

Reply:
> See #398

A welcome change ;-)